src/commctl/cli.py

```diff
@@ -633,6 +636,9 @@ def add_cluster_commands(argument_parser):
     # Sub-command: cluster create
     verb_parser = subject_subparser.add_parser('create')
     verb_parser.required = True
+    verb_parser.add_argument(
+        'type', help='Type of the cluster',
+        choices=('kubernetes', 'host_only'))
```

Comment:
Yes, in fact on further thought since this is not a command one would issue often, I'm okay with either an option or positional arg.  My comment earlier was a bit too knee-jerk.